R/table.R

```diff
@@ -203,14 +203,40 @@ tabledap <- function(x, ..., fields=NULL, distinct=FALSE, orderby=NULL,
   resp <- erd_tab_GET(url, dset = attr(x, "datasetid"), store, callopts)
   loc <- if (store$store == "disk") resp else "memory"
   structure(
-    read_table(resp),
+    format_table(read_table(resp), x),
     class = c("tabledap", "data.frame"),
```

Comment:
@sckott this is the only backwards incompatible change that folks might care about, but it seems pretty harmless...